decidim-core/app/views/decidim/participatory_process_steps/_timeline.html.erb

```diff
@@ -0,0 +1,7 @@
+<ol class="timeline">
+  <% past_step = true %>
+  <% current_participatory_process.steps.each do |step| %>
+    <%= render step, step_status_class: step_class(step, past_step)  %>
+    <% past_step = false if step.active? %>
```

Comment:
Emm... being able to style them accordingly:

![image](https://cloud.githubusercontent.com/assets/1962966/22024296/cf4ae698-dcc9-11e6-8aa7-6cd855d8ed12.png)

And no, it can be `true` or `false`.